cookbooks/rsyncd/recipes/default.rb

```diff
@@ -48,7 +48,8 @@
 systemd_service "rsync-override" do
   service "rsync"
   dropin "override"
-  exec_start "/usr/bin/rsync --daemon --no-detach --bwlimit=16384"
```

Comment:
Was getting rid of the bandwidth limit intentional?

Reply:
yeah it was... Sneaking it in ;-)

Reply:
Happy to make it a separate PR if desired.

Reply:
Removed from PR.